tests/test_app.py

```diff
@@ -8,12 +10,12 @@
 
 @pytest.mark.asyncio
 async def test_missing_env_vars():
-    quart_app = app.create_app()
+    with mock.patch.dict(os.environ, clear=True):
```

Comment:
Now we patch the entire environment, clearing it out, to make sure our tests are isolated from the dev's environment vars. Otherwise this test was sometimes erroring when my env vars were set locally, since this test is meant to emulate the situation of no-env-vars.